apps/http-api-app/src/main/resources/http_api_app.conf

```diff
@@ -28,10 +28,10 @@ application {
         includeRestApi = true
         server = {
             protocol = "http://"
-            host = "localhost"
+            host = "0.0.0.0"
```

Comment:
This would allow any IP address. I don't think that is a good default value. For home network its usually 192.168.0.0 but then lcoalhost would not work. I think its best to pass the value as JVM argument for the non-localhost use cases. Localhost can be considered as a safe default and I think its best to leave it as that.

We need to get anyway Tor working and then we need some way to share the onion address.